platform/ios/src/MGLMapView.h

```diff
@@ -272,23 +273,24 @@ MGL_EXPORT IB_DESIGNABLE
 @property (nonatomic, readonly) UIButton *attributionButton;
 
 /**
- Support for style classes has been removed. This property always returns an empty array.
+ Show the attribution and telemetry action sheet.
+
+ This method is called when the user taps on `attributionButton`. This method is
+ made available so that it can be assigned to custom attribution button
```

Comment:
Nit: added as an action of custom attribution button implementations.

Reply:
→ https://github.com/mapbox/mapbox-gl-native/pull/10085/commits/542d4d8e41637221a87829b83b0539ad80e24a24